docs/output-kafka.asciidoc

```diff
@@ -323,6 +323,13 @@ Kafka down, etc).
 
 A value less than zero is a configuration error.
 
+This plugin will only retry exceptions that are a subclass of
```

Comment:
this is a good start, would also mention that previous plugin versions (<= 10.3.0) kept retrying all errors forever.

Reply:
Aren’t the docs versioned, i.e. previous versions of Logstash docs will not include this new paragraph? Or do you think it’s better to be explicit and add the version number?

Reply:
Yes they are but it's quite uncommon for someone comparing docs side-by-side.
As noted before, the ES plugin still mentions a considerable change in behaviour on handling exceptions:
https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#_retry_policy

Reply:
I've updated the docs to also mention the old behaviour.